opensearch-project/OpenSearch-Dashboards
# [Workspace]Refactor use case selector in workspace creation page

### Description


This PR is for refactoring the use case selector in the workspace creation page. It mainly refactors the features displayed in the use case card. The feature descriptions are moved to a separate use case flyout, which will be displayed after clicking the 'Learn more' button.

## Screenshot


![image](https://github.com/user-attachments/assets/e23dd127-edd1-466f-94ae-64542f672064)
![image](https://github.com/user-attachments/assets/44afe4d4-e049-4cf5-bf54-df2d064abade)




## Testing the changes



- Clone branch code and run `yarn osd bootstrap --single-version loose`
- Add below configs in `config/opensearch_dashboards.yml`
```
savedObjects.permission.enabled: true
workspace.enabled: true
uiSettings:
  overrides:
    'home:useNewHomePage': true
opensearchDashboards.dashboardAdmin.users: ['admin']
```
- Run `yarn start --no-base-path`
- Login with admin user and go to workspace create page by bottom left menu
- The use case selector panel should be displayed like screenshot above
- The flyout should be displayed after "Learn more" button clicked

## Changelog

- feat: [Workspace]Refactor use case selector in workspace creation page

### Check List
